LibCarla/source/carla/rpc/Command.h

```diff
@@ -59,11 +61,17 @@ namespace rpc {
         : description(std::move(description)),
           transform(transform),
           parent(parent) {}
+      SpawnActor(ActorDescription description, const geom::Transform &transform, ActorId parent, const std::string& bone)
+        : description(std::move(description)),
+          transform(transform),
+          parent(parent),
+          socket_name(bone) {}
       ActorDescription description;
       geom::Transform transform;
       boost::optional<ActorId> parent;
       std::vector<Command> do_after;
-      MSGPACK_DEFINE_ARRAY(description, transform, parent, do_after);
+      std::string socket_name;
+      MSGPACK_DEFINE_ARRAY(description, transform, parent, do_after, socket_name);
```

Comment:
Note that new command arguments should be also added to the `Command.cpp` file:
https://github.com/carla-simulator/carla/blob/master/PythonAPI/carla/source/libcarla/Commands.cpp#L79

Reply:
Done